src/cpu/aarch64/acl_convolution_utils.cpp

```diff
@@ -193,7 +194,78 @@ status_t init_conf(acl_conv_gemm_conf_t &acp, memory_desc_t &src_md,
     // clang-format on
 
     // Post-op activations
-    acp.act_info = acl_gemm_convolution_utils::get_acl_act(attr);
+    acp.act_info = acl_convolution_utils::get_acl_act(attr);
+
+    return status::success;
+}
+
+status_t init_conf_gemm(acl_conv_conf_t &acp, memory_desc_t &src_md,
+        memory_desc_t &weights_md, memory_desc_t &dst_md,
+        memory_desc_t &bias_md, const convolution_desc_t &cd,
+        const primitive_attr_t &attr) {
+
+    // General Compute Library checks, memory tags are also set there
+    CHECK(acl_init_conf(acp, src_md, weights_md, dst_md, bias_md, cd, attr));
+
+    // clang-format off
+    // Validate convolution manually to check for return status
+    arm_compute::NEGEMMConvolutionLayer acl_gemm_conv;
+    auto acl_st = acl_gemm_conv.validate(
+        &acp.src_info,
+        &acp.wei_info,
+        acp.with_bias ? &acp.bia_info : nullptr,
+        &acp.dst_info,
+        acp.padstride_info,
+        acp.weights_info,
+        acp.dilation_info,
+        acp.act_info);
+    // clang-format on
+    if (acl_st.error_code() != arm_compute::ErrorCode::OK) {
+        return status::unimplemented;
+    }
+
+    return status::success;
+}
+
+status_t init_conf_wino(acl_conv_conf_t &acp, memory_desc_t &src_md,
+        memory_desc_t &weights_md, memory_desc_t &dst_md,
+        memory_desc_t &bias_md, const convolution_desc_t &cd,
+        const primitive_attr_t &attr) {
+
+    // Under these conditions, fallback to faster GEMM-based convolution
+    // unless the user explicitly specifies Winograd algorithm
+    if (one_of(true, src_md.dims[2] > 112, // ih
+                src_md.dims[3] > 112, // iw
+                src_md.dims[1] < 64, // ic
+                dst_md.dims[1]<64, // oc
+                        dnnl_get_max_threads()> 28)
```

Comment:
nit: formatting

```suggestion
                dst_md.dims[1] < 64, // oc
                dnnl_get_max_threads() > 28)
```

I wonder why clang format didn't catch this...